test/parallel/test-async-wrap-uncaughtexception.js

```diff
@@ -34,13 +34,11 @@ hooks = async_hooks.createHook({
 
 
 process.on('uncaughtException', common.mustCall(() => {
-  assert.strictEqual(call_id, async_hooks.executionAsyncId());
```

Comment:
My guess here is that in certain cases now the async context may or may not be the same. @nodejs/async_hooks folks, I just want to confirm with y'all that this looks correct?

Reply:
Removing these asserts makes this test quite useless. It no longer tests what was tested before.

After this change the random byte comes from the cache therefore no `RANDOMBYTESREQUEST` is done at all.
To keep the behavior of the test as before it's needed to request more random bytes as the cache has.